src/ucp/stream/stream_send.c

```diff
@@ -18,6 +18,13 @@
 #include <ucp/dt/dt.inl>
 
 
+#define UCP_STREAM_SEND_CHECK_STATUS(_ep, _status, _ret, _done) \
+    if (ucs_likely((_status) != UCS_ERR_NO_RESOURCE)) { \
+        _ret = UCS_STATUS_PTR(_status); /* UCS_OK also goes here */ \
+        UCP_EP_STAT_TAG_OP(_ep, EAGER); \
```

Comment:
why TAG?

Reply:
no ideas, @evgeny-leksikov why did you used TAG stat?
@yosefe add stream statistics infra?

Reply:
in this PR, just remove it

Reply:
ok, done